src/Action/Router.php

```diff
@@ -0,0 +1,41 @@
+<?php
+namespace Jahuty\Action;
+
+/**
+ * Routes an action to a path.
+ */
+class Router
+{
+    private static $routes = [
+        Show::class => [
+            'render' => 'snippets/:id/render'
+        ]
+    ];
+
+    public function route(Action $action): string
+    {
+        if (null === ($route = $this->getRoute($action))) {
```

Comment:
yoda you speak!

Reply:
I like the brevity of the `if (null === ($var = some_expression()))` syntax, but it's certainly not the prettiest to read for sure haha. I moved the `Jahuty\Action\Router` to `Jahuty\Uri\Factory` and made lots of changes after your advice on PSR-7 and PSR-17. Thanks my friend!